src/api/apollo.js

```diff
@@ -71,12 +72,25 @@ export default function createApolloClient({
 		// Allow optimizations that are only possible because we have freezeResults=true
 		// see https://github.com/apollographql/apollo-client/pull/4543
 		assumeImmutableResults: true,
+		...(ssr ? {
+		// Set this on the server to optimize queries when SSR
+			ssrMode: true,
+		} : {
+		// This will temporary disable query force-fetching
+			ssrForceFetchDelay: 100,
```

Comment:
This is an interesting alternative to using readQuery to avoid refetching hydrated data, but it sounds like there may be challenges with setting an appropriate delay time. This pr suggests a way to accomplish the same thing without having to rely on timing: https://github.com/apollographql/apollo-client/issues/4814